packages/db/postgres/tables.sql

```diff
@@ -307,11 +307,13 @@ CREATE TABLE IF NOT EXISTS psa_pin_request
   meta            jsonb,
   deleted_at      TIMESTAMP WITH TIME ZONE,
   inserted_at     TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL,
-  updated_at      TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL
+  updated_at      TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL,
+  backup_urls     TEXT[]
```

Comment:
Might be nice if this was `NOT NULL DEFAULT []` so you don't have to distinguish between null and empty.